### PR TITLE
Add hindi translation

### LINKED ISSUE #429: [Docs]: add hindi translation the project
### in which language do you want to translate?

I want to translate documentation to HINDI.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by akkshayTandon**
## Related Issue

closes #429

## Description

Added the HINDI translations for the following
- README.md
- contributing.md
- CODE_OF_CONDUCT.md

## Screenshots
github

### CODE REVIEW

**Comment by priyankarpal** [comment]
@akkshayTandon also, update the available language in the docs

**Comment by akkshayTandon** [comment]
@priyankarpal I am having trouble cloning it locally. Whenever I try to run `npm i` it gets me error saying `unable to resolve dependency tree`, `Fix the upstream dependency conflict or retry this command with --force or --legacy-peer-deps`.

**Comment by priyankarpal** [comment]
> @priyankarpal I am having trouble cloning it locally. Whenever I try to run `npm i` it gets me error saying `unable to resolve dependency tree`, `Fix the upstream dependency conflict or retry this command with --force or --legacy-peer-deps`.

use `pnpm i` not `npm i` we are using `pnpm`

File: .env.example
Comment (priyankarpal) [inline .env.example:1]: don't delete this file

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: .env.example
Comment (akkshayTandon) [reply .env.example:1]: Oh! I got it, I renamed the file as per the instructions. I think we should add this step as well in the docs,`to rename the .env file back to .env.example after the commit is done?

Comment (priyankarpal) [reply .env.example:1]: already have it in docs. just add `.env.example` file you deleted this file

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**